statuspage/index.html

```diff
@@ -5,7 +5,7 @@
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<script src="js/d3.v3.min.js" charset="utf-8"></script>
-		<script src="js/s3.js"></script>
+		<script src="js/fs.js"></script>
```

Comment:
This changes the default storage provider to the local file system; if we do this, the docs should probably be rewritten to accommodate. Maybe best for a separate PR.... so maybe we should change this back for now.